### PR TITLE
Changed map dropdown UI

### PR DESCRIPTION
**Pull request by kofimeighan**
image

### CODE REVIEW

**Review by Jaxobc — APPROVED** [review]

**Comment by Jaxobc** [comment]
This looks great Kofi. I'll be asking you for help with my dropdown!

File: portfolio/src/main/webapp/statistics.html [hunk 1]
```diff
@@ -118,26 +118,12 @@ <h4>Leave a marker on the map for protests you will be attending!</h4>
                   id="name"
                   name="name"
                   type="text"
-                  placeholder="Your Name *"
+                  placeholder="Name of protest *"
```
Comment (briafassler9) [inline portfolio/src/main/webapp/statistics.html]: nit: capitalize protest

File: portfolio/src/main/webapp/statistics.html [hunk 2]
```diff
@@ -67,38 +67,38 @@ <h3 class="section-subheading text-muted">
           <div class="dropdown">
             <button
               type="button"
-              class="btn btn-primary dropdown-toggle"
+              class="btn btn-secondary dropdown-toggle"
               data-toggle="dropdown"
               aria-haspopup="true"
               aria-expanded="false"
             >
               Where martyrs have died
```
Comment (briafassler9) [inline portfolio/src/main/webapp/statistics.html]: Not all martyrs we have featured died, for example Justin Howell was just severely injured. Maybe think about changing the name displayed on the button.

File: portfolio/src/main/webapp/js/custom-script.js [hunk 3]
```diff
@@ -53,8 +53,8 @@ function statisticsOnLoad() {
     ['Robert Fuller', 'Palmdale, California'],
     ['James Scurlock', 'Omaha, Nebraska'],
     ['Elijah McClain', 'Aurora, Colorado'],
-    ['Placeholder', 'Mountain View, CA'],
-    ['Placeholder', 'Mountain View, CA'],
+    ['Justin Howell', 'Austin, Texas'],
+    ['Jamel Floyd', 'Brooklyn, New York'],
   ];
 
   const iconicProtestData = [
```
Comment (briafassler9) [inline portfolio/src/main/webapp/js/custom-script.js:59]: For all options, make sure everything is capitalized other than certain articles like a, and, the, etc. Also, only one has the date so I'd either add the date for the other options or remove it for consistency.

File: portfolio/src/main/webapp/statistics.html (hunk 1 above)
Comment (briafassler9) [reply portfolio/src/main/webapp/statistics.html]: nit: same for "Location of protest" and "Description of protest"

**Comment by briafassler9** [comment]
Adding another section like "Date of Protest" for user-submitted protests can be helpful to users who come across our website to know if the date for a protest is coming up or already passed.

File: portfolio/src/main/webapp/statistics.html (hunk 2 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/statistics.html]: I think i'm going to remove Justin Howell from the list to keep the weight and significance of the list

**Comment by kofimeighan** [comment]
> Adding another section like "Date of Protest" for user-submitted protests can be helpful to users who come across our website to know if the date for a protest is coming up or already passed.

@briafassler9 I'm planning on adding a description to each pin so that you will be able to see when the protest was.

File: portfolio/src/main/webapp/statistics.html (hunk 2 above)
Comment (briafassler9) [inline portfolio/src/main/webapp/statistics.html]: nit: capitalize all words

File: portfolio/src/main/webapp/statistics.html [hunk 4]
```diff
@@ -118,26 +118,12 @@ <h4>Leave a marker on the map for protests you will be attending!</h4>
...
-                  required
-                  data-validation-required-message="Please enter your phone number."
-                />
-                <p class="help-block text-danger"></p>
-              </div>
-            </div>
-            <div class="col-md-6">
               <div class="form-group">
                 <input
                   class="form-control"
```
Comment (briafassler9) [inline portfolio/src/main/webapp/statistics.html:129]: nit: for the location placeholder, capitalize protest

**Review by wangtong-g — APPROVED** [review]

**Review by briafassler9 — APPROVED** [review]

**Pull request merged**